vitrina/orgs/views.py

```diff
@@ -8,3 +9,19 @@ class OrganizationListView(ListView):
     queryset = Organization.public.order_by('-created')
     template_name = 'vitrina/orgs/list.html'
     paginate_by = 20
+
+
+class OrganizationSearchResultsView(ListView):
+    model = Organization
+    template_name = 'vitrina/orgs/list.html'
+    paginate_by = 20
+    
+    def get_queryset(self):
+        query = self.request.GET.get('q')
+        if query:
+            orgs = Organization.objects.filter(
+                Q(title__icontains=query)
```

Comment:
Use of `Q` is redundant here. You can simply do `filter(title__icontains=query)`. And also, use `public` manager.